content/docs/18-knowledgebase/00-how-to/2-palette-sso-with-adfs.md

```diff
@@ -0,0 +1,222 @@
+
+
+In the next screen, define the group claim as desired. In this example, we have a group in Active Directory called `SpectroTeam - Admins` and we want anyone that is a member of that group, to be issued a `groups` claim with the value `Admins`. In Palette this user will then automatically be mapped to a group with the same `Admins` name. We can then assign RBAC permissions to that group in Palette to give it the desired access:
+
+![Set Group Claim](palette-sso-with-adfs-images/18-set_group_claim.png)
+
+
+Click **Finish** to add the rule. Click **OK** to save the changes to the Web API rule and click **OK** again to save the changes to the Application Group. Finally, note your AD FS identifier, we will need this for Palette in the next step. Typically this is your AD FS name plus `/adfs`. You can also take the Federation Service identifier and remove `/services/trust` from that URL:
+
+![Note AD FS Name](palette-sso-with-adfs-images/19-note_adfs_name.png)
+
+
+
+## 2 - Enable OIDC SSO in Palette
+
+Open a web browser and navigate to your Palette subscription. Navigate to **Tenant Settings** --> **SSO** and click **OIDC**. Enter the following information:
+* Issuer URL: Your AD FS issuer URL, typically this is your AD FS name plus `/adfs`
+* Client ID: the Client Identifier that we noted down in the beginning of the previous step
+* Client Secret: the Shared Secret that we generated in the previous step
+* Default Teams: leave empty if you don't want users without any groups claims to be assigned to a default group. If you do, enter the desired default group name here. If you use this option, be careful with how much access you give to that group.
+* Scopes: set this to `openid` and `allatclaims`
+* Email: leave at the default of `email`
+* First Name: leave at the default of `given_name`
+* Last Name: leave at the default of `family_name`
+* Spectro Team: leave at the default of `groups`
+
+![Enable Palette OIDC SSO](palette-sso-with-adfs-images/20-configure_palette_oidc.png)
+
+
+When all the information has been entered, click the **Enable** button at the button to enable SSO. You should receive an `OIDC configured successfully` message.
+
+
+## 3 - Create Teams in Palette
+
+All that's left to do now is to create Team(s) in Palette for the groups claim(s) that were configured in AD FS, and give them the appropriate permissions. For this example, we will create the `Admins` team and give it `Tenant Admin` permissions. You can repeat this for any other team that you've configured group claims for.
+
+Open a web browser and navigate to your Palette subscription. Navigate to **Tenant Settings** --> **Users & Teams** --> **Teams** tab. Click the **+ Create Team** button.
+
+![Create Palette Team](palette-sso-with-adfs-images/21-create_team.png)
+
+
+Specify `Admins` in the **Team name** field. We don't need to set any members now, as this will happen automatically from the SSO. Click **Confirm** to create the team.
```

Comment:
```suggestion
Specify `Admins` in the **Team name** field. You don't need to set any members now, as this will happen automatically from the SSO. Click **Confirm** to create the team.
```